python/ray/data/tests/test_dataset.py

```diff
@@ -2883,6 +2883,8 @@ def test_groupby_arrow(ray_start_regular_shared):
         lambda r: r["value"] > 10).groupby("value").count()
     assert agg_ds.count() == 0
 
+
+def test_groupby_arrow_count(ray_start_regular_shared):
```

Comment:
Thanks for splitting these up.

Reply:
Yep, the mono-test was getting a bit slow!